### ISSUE TITLE
create 3d curb at intersections

### ISSUE THREAD

**Issue by kfarr** [3DStreet/3dstreet#231]
in the same manner that sidewalks on street component now have depth, we should add the same depth (using a box geometry) to the intersection sidewalk / curb material.

current:
![image](https://user-images.githubusercontent.com/470477/215620411-e3beac48-cf84-4133-ab92-4c73ea0f54b5.png)

from https://github.3dstreet.org/intersection.html#https://streetmix.net/kfarr/3

Instead, the sidewalks on intersection component should have depth (and ideally always correct aspect ratio regardless of height / width -- but we can do that later)

**Comment by kfarr** [comment]
Question: These colored boxes are raised sidewalks? Maybe it's better to make a trapezoidal geometry here?  I found that trapezoidal geometry can be created from cilynder geometry
Answer: No instead those colored boxes are an error. This is a ticket about the corner curbs, not the sidewalks. Instead of having a plane to represent a sidewalk / corner curb, instead to have a box in the same fashion that we did for a sidewalk segment of a street.
![image](https://user-images.githubusercontent.com/470477/221018604-0656daf5-0071-45fe-bbb5-48e2fe907355.png)
